Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -494,6 +494,7 @@ def setScores(self, scores, labels, values, colors, rownames=None):
         self.__setup()
 
     def setRowNames(self, names):
+        ROW_NAMES_WIDTH = 200
```

Comment:
There are some widgets that have all their settings (like widths, various colors...) collected as constants at the top, which is convenient when you want to improve its design. Moving a single constant is the first step in this direction. :)